clans/src/main/java/me/mykindos/betterpvp/clans/clans/commands/subcommands/ListSubCommand.java

```diff
@@ -0,0 +1,118 @@
+package me.mykindos.betterpvp.clans.clans.commands.subcommands;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import me.mykindos.betterpvp.clans.clans.Clan;
+import me.mykindos.betterpvp.clans.clans.ClanManager;
+import me.mykindos.betterpvp.clans.clans.ClanRelation;
+import me.mykindos.betterpvp.clans.clans.commands.ClanCommand;
+import me.mykindos.betterpvp.clans.clans.commands.ClanSubCommand;
+import me.mykindos.betterpvp.core.client.Client;
+import me.mykindos.betterpvp.core.command.SubCommand;
+import me.mykindos.betterpvp.core.components.clans.data.ClanMember;
+import me.mykindos.betterpvp.core.gamer.GamerManager;
+import me.mykindos.betterpvp.core.utilities.UtilMessage;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import javax.inject.Named;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Stream;
+
+@Singleton
+@SubCommand(ClanCommand.class)
+public class ListSubCommand extends ClanSubCommand {
+
+    @Inject
+    public ListSubCommand(ClanManager clanManager, GamerManager gamerManager) {
+        super(clanManager, gamerManager);
+        aliases.addAll(List.of("?", "h"));
```

Comment:
Probably want to remove these aliases since "?" and "h" are usually recognized as part of a help action.

Reply:
Unintended